website/docs/language/resources/provisioners/connection.mdx

```diff
@@ -113,13 +113,13 @@ indirectly with a [bastion host](https://en.wikipedia.org/wiki/Bastion_host).
 | `bastion_private_key` | The contents of an SSH key file to use for the bastion host. These can be loaded from a file on disk using [the `file` function](/terraform/language/functions/file). | The value of the `private_key` field. |
 | `bastion_certificate` |  The contents of a signed CA Certificate. The certificate argument must be used in conjunction with a `bastion_private_key`. These can be loaded from a file on disk using the [the `file` function](/terraform/language/functions/file). |
 
-## Connection through a HTTP Proxy with SSH
+## Connection through a HTTP/SOCKS5 Proxy with SSH
 
-The `ssh` connection also supports the following fields to facilitate connections by SSH over HTTP proxy.
+The `ssh` connection also supports the following fields to facilitate connections by SSH over HTTP or SOCKS5 proxy.
 
 | Argument | Description | Default |
 |---------------|-------------|---------|
-| `proxy_scheme` | http or https | |
+| `proxy_scheme` | http, https, socks5 | |
```

Comment:
```suggestion
| `proxy_scheme` | You can specify one of the following values: `http`, `https`, `socks5` | |
```
I _think_ the intention is to describe what values to set for this param. If not, then HTTP and HTTPS should be uppercase because they are proper acronyms. SOCKS5 also appears to be spelled in all caps, but that appears to be for branding purposes.

Reply:
Correct these are values.